test/metabase/api/dashboard_test.clj

```diff
@@ -383,7 +385,20 @@
                                             :cache_ttl     1234
                                             :creator_id    (mt/user->id :rasta)
                                             :collection_id true})
-                 (dashboard-response (db/select-one Dashboard :id dashboard-id)))))))))
+                 (dashboard-response (db/select-one Dashboard :id dashboard-id)))))
+
+        (testing "No-op PUT: Do not return 500"
+          (t2.with-temp/with-temp [Card          {card-id :id} {}
+                                   DashboardCard dashcard      {:card_id card-id, :dashboard_id dashboard-id}]
```

Comment:
I think people will be happy with the new syntax for `with-temp`. No more `with-temp` vs. `with-temp*`